components/application-operator/charts/application/templates/deployment.yaml

```diff
@@ -136,8 +136,4 @@ spec:
         ports:
         - containerPort: {{ .Values.eventService.deployment.args.externalAPIPort }}
           name: http-api-port
-        {{- if .Values.eventService.deployment.resources }}
-        resources:
-        {{- toYaml .Values.eventService.deployment.resources | nindent 10 }}
-        {{- end }}
 {{- end }}
```

Comment:
Following are my concerns:
- This change comes with an image change as well. I’d include the change in image as well in the same PR so that the context is preserved.
- Values still exist [here](https://github.com/kyma-project/kyma/blob/master/components/application-operator/charts/application/values.yaml#L81) which need to be removed as well.